ci-templates/stages.yml

```diff
@@ -15,7 +15,7 @@ stages:
     jobs:
       - job: Cache_Maven_Repo
         displayName: 'Linux Maven Repo'
-        condition: and(eq(variables.LINUX_USE_VMS, ${{parameters.expectUseVMs}}),succeeded()) 
+        condition: and(eq(variables.LINUX_USE_VMS, ${{parameters.expectUseVMs}}),succeeded())
```

Comment:
I don't mind the reformating, but that creates noises.

Reply:
I agree, must be my IDE. I'll revert.

Reply:
done. Thanks for catching this one.